### ISSUE TITLE
bpf_attach_kprobe() signature changed in libbcc v0.6.0

### ISSUE THREAD

**Issue by jwhited** [iovisor/gobpf#132]
```
vendor/github.com/iovisor/gobpf/bcc/module.go:212: too many arguments in call to _C2func_bpf_attach_kprobe
	have (_Ctype_int, uint32, *_Ctype_char, *_Ctype_char, number, number, number, nil, nil)
	want (_Ctype_int, uint32, *_Ctype_char, *_Ctype_char, _Ctype_ulong)
```

**Comment by bobrik** [comment]
This should be fixed with #130.

**Comment by schu** [comment]
> This should be fixed with #130.

Yes, update is in master now. Thanks.
